sycl/include/sycl/handler.hpp

```diff
@@ -1663,21 +1669,24 @@ class __SYCL_EXPORT handler {
   }
 
   template <typename KernelName = detail::auto_name, typename KernelType>
-  void parallel_for(range<1> NumWorkItems, _KERNELFUNCPARAM(KernelFunc)) {
+  void parallel_for(range<1> NumWorkItems ANNOTATE(range),
```

Comment:
Do we also need to annotate the equivalent `queue` shortcut functions or do they pass through here anyway?

Reply:
Those go through the handler ones, yep